### ISSUE TITLE
Check driver jobs builder image

### ISSUE THREAD

**Issue by leodido** [falcosecurity/test-infra#228]
This is what the `build-drivers.sh` script has:

https://github.com/falcosecurity/test-infra/blob/bf7113ac58b2411d1b0e6a7c6c109de53ad8a44c/images/build-drivers/build-drivers.sh#L53-L57

While this is what I see in `build-drivers-*` periodic jobs:

```
******************************************************
Ran DriverKit tests
******************************************************
{"component":"entrypoint","file":"prow/entrypoint/run.go:247","func":"k8s.io/test-infra/prow/entrypoint.gracefullyTerminate","level":"error","msg":"Process gracefully exited before 10m0s grace period","severity":"error","time":"2020-11-25T01:11:28Z"}
{"component":"entrypoint","error":"process timed out","file":"prow/entrypoint/run.go:80","func":"k8s.io/test-infra/prow/entrypoint.Options.Run","level":"error","msg":"Error executing test process","severity":"error","time":"2020-11-25T01:11:28Z"}
```

From [this](https://prow.falco.org/log?job=build-drivers-amazonlinux-periodic&id=1331357616768552960) prowjob logs (scroll to the end).

This means the ProwJob is using a different image.

**Comment by maxgio92** [comment]
Can we consider this fixed @jonahjon @leodido, WDYT?

**Comment by maxgio92** [comment]
Closed by #249
